packages/datadog-plugin-aws-sdk/test/eventbridge.spec.js

```diff
@@ -87,7 +87,45 @@ describe('EventBridge', () => {
       parentId = '0000000000000000'
       eventbridge.requestInject(span.context(), request, tracer)
 
-      expect(request.params).to.deep.equal({ 'Entries': [{ 'Detail': '{"custom":"data","for":"my users","from":"Aaron Stuyvenberg","_datadog":{"x-datadog-trace-id":"456853219676779160","x-datadog-parent-id":"456853219676779160","x-datadog-sampling-priority":"1"}}' }] })
+      const cleaned = request.params.Entries[0].Detail.replace(/"ms":\d+/g, '"ms":000')
+      request.params.Entries[0].Detail = cleaned // replace the miliseconds with 000
```

Comment:
Why?

Reply:
The value of `ms` in these tests will vary based on what time the test is run.